File: c++/problem-13(FindingShoe)/FINDSHOES.cpp
#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t;
	cin>>t;
	while (t--){
	    int n,L;
	    cin>>n>>L;
	    if (n<L){
	       int total=n;
	       cout<<total<<endl;
	    }
	    else {
	        int r=n-L;
	        int totalR = 2*r;
	        int total_shoe = totalR + L;
	        cout<<total_shoe<<endl;
	    }
	}
	return 0;
}

File: python/problem-14(FindingShoe)/FINDSHOES.py
# cook your dish here
t=int(input())
for i in range(t):
    (n,L) = map(int,input().split())
    if (n<L):
        total=n
        print(n)
    else :
        r=n-L
        totalR=2*r
        total_shoe = totalR+L
        print(total_shoe)
                
        
        